helix-term/src/commands.rs

```diff
@@ -2715,10 +2715,15 @@ fn push_jump(view: &mut View, doc: &Document) {
 }
 
 fn goto_line(cx: &mut Context) {
-    goto_line_impl(cx.editor, cx.count)
+    if cx.count.is_some() {
+        goto_line_without_jumplist(cx.editor, cx.count);
+
+        let (view, doc) = current!(cx.editor);
+        push_jump(view, doc);
```

Comment:
`push_jump` is still being called before `goto_line_without_jumplist`. This will messup the jumplist. Simply changing this to:

```suggestion
        let (view, doc) = current!(cx.editor);
        push_jump(view, doc);
        
        goto_line_without_jumplist(cx.editor, cx.count);
```

should work

Reply:
Ah, good catch. I forgot I had updated this as well.